lib/mayaUsd/python/wrapMeshWriteUtils.cpp

```diff
@@ -59,12 +59,12 @@ _GetMeshNormals(const std::string& meshDagPath)
         return make_tuple(normalsArray, interpolation);
     }
 
-    UsdMayaMeshUtil::GetMeshNormals(meshObj, &normalsArray, &interpolation);
+    UsdMayaMeshReadUtils::getMeshNormals(meshObj, &normalsArray, &interpolation);
 
     return make_tuple(normalsArray, interpolation);
 }
 
-// Dummy class for putting UsdMayaMeshUtil namespace functions in a Python
+// Dummy class for putting UsdMayaMeshReadUtils namespace functions in a Python
 // MeshUtil namespace.
 class DummyScopeClass{};
 
```

Comment:
Ideally, the `wrapMeshUtil()` function below would be renamed to `wrapMeshWriteUtils()` as well to match the file name, which would also require a change in the module.cpp.

Reply:
@mattyjams  see 45f0f518a5ca86932203fc092ac53084eaaf00d7